app/helpers/complaints_helper.rb

```diff
@@ -0,0 +1,27 @@
+module ComplaintsHelper
+  FORMATTED_STATUS = {
+    0 => "In Progress",
+    1 => "Closed",
+    2 => "Rec. Closure",
+    3 => "Rec. Reopening",
+    4 => "Reopened"
+  }
+
+  STATUS_SORT_ORDER = {
+    "New" => 0,
+    "Rec. Reopening" => 1,
+    "Reopened" => 2,
+    "In Progress" => 3,
+    "Rec. Closure" => 4,
+    "Closed" => 5
+  }
+
+  def status(complaint_attributes)
```

Comment:
[thought] Eventually, I think we're going to want a model for complaints, where this would be a method. I think it makes sense to punt that until after getting the real first API though.

Reply:
possibly a decorator lol too soon to think that way davida

Reply:
lol - I mean, yeah, the model I'm thinking of would in fact be more of a decorator than anything else.